src/System.Threading.Tasks/src/System.Threading.Tasks.csproj

```diff
@@ -27,7 +28,6 @@
     <Compile Include="System\Threading\Tasks\TaskExtensions.CoreRT.cs" />
     <ProjectReference Include="..\..\System.Runtime\src\redist\System.Runtime.depproj" />
     <TargetingPackReference Include="System.Private.CoreLib" />
-    <TargetingPackReference Include="System.Private.CoreLib.Threading" />
```

Comment:
Ah.. looks like that build configuration doesn't even exist in the .builds file so we can probably remove all these or convert them to uap101 configuration and add that configuration in the .builds file.